### ISSUE TITLE
ha-entity doesn't provide option to change name of entity in non-hassio setup

### ISSUE THREAD

**Issue by nidayand** [zachowj/node-red-contrib-home-assistant-websocket#373]
### Describe the bug

The problem is that I don't get the same possibilities to work with entities in Home Assistant in the standalone version of Node-RED as with the Hassio version of Node-RED. I cannot set the name of an entity nor work with the attributes as per the configuration settings when running in Hassio.

I'm running hassio with Node-RED as well as Node-RED in multiple instances. And additionally I've tested to create a completely new instance of Node-RED on Docker just to make sure that the error isn't related to other components.
I don't know if the below is related to the issue but it is something that I've spotted that differs from the Hassio setup and a standalone Node-RED instance.

### To Reproduce

Pre-condition: 
- An Home Assistant instance with the Node-RED plugin installed.
- Home Assistant has been restarted

1. Pull latest docker container 
`docker pull nodered/node-red:latest`
2. Install latest version of node-red in container 
`docker run -it -p 1880:1880 --name mynodered nodered/node-red:latest`
3. Verify version
```
Welcome to Node-RED
===================

13 Apr 11:20:19 - [info] Node-RED version: v1.3.1
13 Apr 11:20:19 - [info] Node.js  version: v10.24.1
```
2. Add node-red-contrib-home-assistant-websocket via the palette
3. Add an entity node to the flow
4. Open Chrome Developer tools
5. Open the added entity node for configuration
6. Console log reads
```
ReferenceError: addTitle is not defined
    at t.<computed>.<computed>._create (red.min.js:16)
    at t.<computed>.<computed>._create (vendor.js:10)
    at t.<computed>.<computed>._createWidget (vendor.js:10)
    at new t.<computed>.<computed> (vendor.js:10)
    at HTMLOListElement.<anonymous> (vendor.js:10)
    at Function.each (vendor.js:2)
    at s.fn.init.each (vendor.js:2)
    at s.fn.init.t.fn.<computed> [as editableList] (vendor.js:10)
    at Object.oneditprepare (<anonymous>:1:1808)
    at r (red.min.js:16)
```
7. I don't get the option to modify parameters such as the entity name

### Expected behavior

As per Hassio instance with Node-RED I get a lot of more options to configure the entity such as the name and attributes.
![Screenshot 2021-04-13 at 13 25 44](https://user-images.githubusercontent.com/2181965/114545192-f584a500-9c5b-11eb-9f84-a316d99bd80a.png)

### Screenshots

**Node-RED output as per guide above**
There are no way to modify parameters such as name or attributes of the Home assistant entity
![Screenshot 2021-04-13 at 13 25 10](https://user-images.githubusercontent.com/2181965/114545371-29f86100-9c5c-11eb-824a-217390073586.png)

**Same as above but WITH Home Assistant configured**
Note that I've restarted Home Assistant. I've deployed the flow with the configuration so it is saved serverside.
![Screenshot 2021-04-13 at 13 30 30](https://user-images.githubusercontent.com/2181965/114545619-75ab0a80-9c5c-11eb-814c-a987b3515a08.png)

### Environment (please complete the following information):

- Node Red Version: 1.3.1
- Node.js Version: 10.24.1
- NR Home Assistant Plugin Version: 0.30.1
- Is Node Red running in Docker: yes

### Other (please complete the following information):

- Have you searched previous issues for duplicates?: Yes

### Additional context
N/A

**Comment by zachowj** [comment]
You can follow the issue here https://github.com/node-red/node-red/issues/2933

**Comment by zachowj** [comment]
Fixed in Node-RED 1.3.2

**Comment by nidayand** [comment]
Good stuff. Thanks for the feedback
